tests/ci.benchmarks/requirements.txt

```diff
@@ -1 +1 @@
-redisbench_admin>=0.3.1
+redisbench_admin>=0.4.0
```

Comment:
Would we like those requirements to be installed as part of `system-setup.py`?

Reply:
given it has an extended deps list I didn't want to break automation in any way because of this. It's documented on the Readme how to install it.